students/jim/Lesson03/mailroom.py

```diff
@@ -70,5 +65,14 @@ def create_report():
     # Back to the  menu
     init_script()
 
+
+def print_letter(donor_name, donation_amt):
+    print("Dear {}:\n".format(donor_name))
+    print("Thank you so much for your kind donation of {} dollars.".format(donation_amt))
+    print("It will be put to great use. Give yourself a big pat on the back.\n")
+    print("Sincerely,")
+    print("The Management.")
+    return
+
```

Comment:
👍

Reply:
You could also wrap this into a raw docstring: 

    r'''Dear {}:
    Now I am printing some stuff...
    And more stuff...
    '''